presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -199,7 +218,7 @@ public boolean isLimitGuaranteed(ConnectorSession session)
     public void commitCreateTable(ConnectorSession session, JdbcOutputTableHandle handle)
     {
         delegate.commitCreateTable(session, handle);
-        invalidateTablesCaches();
+        invalidateTablesCaches(toTableHandle(handle));
```

Comment:
This should be separate commit, as this does not "fix cache invalidation" -- the invalidation here was correct.

In fact, I think this is an unnecessary complexity and i would rather not have this change.

(same for other table/schema level invalidations)